test/TritonGPU/amd/in-thread-transpose.mlir

```diff
@@ -319,6 +319,55 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 8 : i32, ttg.targ
 
 // -----
 
+// Test that backward SCF traversal correctly process nested CF structures
```

Comment:
The purpose of this test should be focusing on checking we can apply the transformation without a leading load op. So the nested conditions copied from the previous commit is irrelevant and distracting? We should be able to simplify the test here to be more orthogonal to other tests?